pyproject.toml

```diff
@@ -29,7 +29,7 @@ boto3 = "^1.28.46"
 moto = "^4.2.2"
 types-sqlalchemy = "^1.4.53.38"
 urllib3 = "^1.26.17"
-db-client = { git = "https://github.com/climatepolicyradar/navigator-db-client.git", tag = "v3.8.5" }
+db-client = { git = "https://github.com/climatepolicyradar/navigator-db-client.git", rev = "b92c0a9815c2f36af05409c2904935e351d8c84f" }
```

Comment:
Assume this will be updated before merge - just resolve this once done :smile:

Reply:
Done